BackofficeBundle/Resources/translations/messages.fr.yml

```diff
@@ -372,7 +372,7 @@ open_orchestra_backoffice:
             content_type:
                 list: Type de contenu
             status:
-                list: Etapes de workflow
+                list: Etape de workflow
```

Comment:
This is a global problem: take the translation above and below: ``Type de contenu`` and ``UtilisateurS``

Maybe you should create a ticket to choose one time for all